src/bci_build/package/postgres/README.md.j2

```diff
@@ -0,0 +1,114 @@
+
+#### POSTGRES_PASSWORD
+
+This environment variable is required for you to use the PostgreSQL image. It must not be empty or undefined. This environment variable sets the superuser password for PostgreSQL. The default superuser is defined by the `POSTGRES_USER` environment variable.
+
+#### POSTGRES_USER
+
+This optional environment variable is used in conjunction with `POSTGRES_PASSWORD` to set a user and its password. This variable will create the specified user with superuser power and a database with the same name. If it is not specified, then the default user of `postgres` will be used.
+
+#### POSTGRES_DB
+
+This optional environment variable can be used to define a different name for the default database that is created when the image is first started. If it is not specified, then the value of `POSTGRES_USER` will be used.
+
+#### POSTGRES_INITDB_ARGS
+
+This optional environment variable can be used to send arguments to `postgres initdb`. The value is a space-separated string of arguments as `postgres initdb` would expect them. This is useful for adding functionality like data page checksums: `-e POSTGRES_INITDB_ARGS="--data-checksums"`.
+
+#### POSTGRES_INITDB_WALDIR
+
+This optional environment variable can be used to define another location for the Postgres transaction log. By default, the transaction log is stored in a subdirectory of the main Postgres data folder (`PGDATA`). Sometimes it can be desirable to store the transaction log in a different directory which may be backed by storage with different performance or reliability characteristics.
+
+#### POSTGRES_HOST_AUTH_METHOD
+
+This optional variable can be used to control the `auth-method` for host connections for all databases, all users, and all addresses. If unspecified then `scram-sha-256` password authentication is used.
+
+On an uninitialized database, this will populate `pg_hba.conf` via this approximate line:
+
+`echo "host all all all $POSTGRES_HOST_AUTH_METHOD" >> pg_hba.conf`
+
+See the PostgreSQL documentation on [password Authentication](https://www.postgresql.org/docs/14/auth-password.html) and [pg_hba.conf](https://www.postgresql.org/docs/14/auth-pg-hba-conf.html) for more information about possible values and their meanings.
+
+**Note 1:** If you set `POSTGRES_HOST_AUTH_METHOD` to `trust`, then `POSTGRES_PASSWORD` is not required since it allows anyone to connect without a password.
+
+**Note 2:** If you set this to an alternative value, you might need additional `POSTGRES_INITDB_ARGS` for the database to initialize correctly.
+
+#### PGDATA
+
+The default value for this variable is `/var/lib/pgsql/data`. This optional variable can be used to define another location for the [database files](https://www.postgresql.org/docs/current/storage-file-layout.html).
+
+This is an environment variable that is not container specific. The entrypoint script takes it into account because the variable is used by the `postgres` server binary.
```

Comment:
```suggestion
This is an environment variable is not container-specific. The entrypoint script takes it into account, because the variable is used by the `postgres` server binary.
```